githubJingyan-Fuquay.md

```diff
@@ -0,0 +1,5 @@
+## Fuqary Varina, NC
```

Comment:
@zhaojingyan1    Are you from Fuquay also?   Can you update this to reflect your hometowm?